README.md

```diff
@@ -20,7 +20,7 @@ mdcat in [WezTerm], with "One Light (base16)", "Gruvbox Light", and "Darcula
 
 ## Features
 
-`mdcat` works best with [iTerm2], [WezTerm], and [kitty], and a good terminal font with italic characters.
+`mdcat` works best with [iTerm2], [WezTerm], [kitty], and [Ghostty], and a good terminal font with italic characters.
```

Comment:
As a concession to my bad impression about the tool, let's not sort of advertise it here, at least not until it's really open source 🙂

Let's just add it to the table and call it a day 🙂

Reply:
Removed, left in the table.